package.json

```diff
@@ -3,75 +3,71 @@
   "private": true,
   "appName": "BCRS Shared Components",
   "scripts": {
-    "storybook": "start-storybook -p 6006",
-    "storybook:build": "build-storybook -o ./docs",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern",
-    "test:unit-silent": "vue-cli-service test:unit --silent",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "storybook": "storybook dev",
+    "build-storybook": "storybook build -o ./docs",
+    "lint": "eslint . --ext js,ts,vue",
+    "test:unit": "vitest --testPathPattern",
+    "test:coverage": "vitest run --coverage"
```

Comment:
Note: coverage is not configured (same as before).

Ref: https://vitest.dev/config/#coverage